src/config.ts

```diff
@@ -1,7 +1,23 @@
-import createDebug from 'debug';
-import { join as joinPath } from 'path';
-
-const configDebug = createDebug('superface:config');
+import { IEnvironment } from './lib/environment';
+import { IFileSystem } from './lib/io';
+import { NodeFileSystem } from './lib/io/filesystem.node';
+import { ILogger, LogFunction } from './lib/logger/logger';
+
+const DEBUG_NAMESPACE = 'config';
+
+type JoinPath = { path: { join: IFileSystem['path']['join'] } };
```

Comment:
question: Why not just request `IFileSystem` directly?

Reply:
Because this is used outside of SDK and having other users implement the entire FileSystem seems like overkill.. but the filesystem dependency will be removed soon :tm: anyway hopefully